Resources/Changelog/Changelog.yml

```
@@ -11595,3 +11595,558 @@ Entries:
   id: 6837
   time: '2025-02-11T19:26:32.0000000+00:00'
   url: https://github.com/Simple-Station/Einstein-Engines/pull/1753
+- author: EctoplasmIsGood
+  changes:
+    - type: Fix
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)